cllc-public-app/ClientApp/src/app/models/contact.model.ts

```diff
@@ -5,7 +5,7 @@ export class Contact {
   lastname: string;
   emailaddress1: string;
   telephone1: string;
-  address1_street1: string;
+  address1_line1: string;
```

Comment:
Are changes to the Angular code required for this?

Reply:
no i did a global search. this was the only occurrence of `address1_street1` must have been a typo that didn't effect anything..

Reply:
and when i say didn't effect anything i mean theres a chance something is relying on the incorrect behavior that was in place before..